drivers/overlay/joinleave.go

```diff
@@ -145,6 +145,21 @@ func (d *driver) Join(nid, eid string, sboxKey string, jinfo driverapi.JoinInfo,
 	return nil
 }
 
+func (d *driver) DecodeTableEntry(tablename string, key string, value []byte) (string, string) {
+	if tablename != ovPeerTable {
+		logrus.Errorf("DecodeTableEntry: unexpected table name %s", tablename)
+		return "", ""
+	}
+
+	var peer PeerRecord
+	if err := proto.Unmarshal(value, &peer); err != nil {
+		logrus.Errorf("DecodeTableEntry: failed to unmarshal peer record: %v for key %s", err, key)
+		return "", ""
+	}
+
+	return key, "Host IP " + peer.TunnelEndpointIP
```

Comment:
IMO we should separate the property from the value with a semicolon
`return key, "Host IP: " + peer.TunnelEndpointIP`